internal/rpc/client.go

```diff
@@ -203,16 +203,6 @@ func (c *Client) Call(ctx context.Context, facade string, version int, id, metho
 
 	select {
 	case <-ch:
-		permissionsRequired, err := checkPermissionsRequired(ctx, *respMsg)
```

Comment:
The `rpc` package was using `checkPermissionsRequired` which now lives in `rpcproxy.go`. I've removed this as there is no need to check for the permissionsRequired error here.